includes/wikia/models/DesignSystem/WdsImageTrait.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+trait WdsImageTrait {
+	public $image;
+
+	public function setSvgImageData( $key ) {
+		$this->image = $key;
+		$this->{'image-data'} = new WdsSvg( $key );
```

Comment:
Instead of "hacking" PHP with variable names with dashes what do you think about using [JsonSerializable](http://nl3.php.net/manual/en/jsonserializable.jsonserialize.php) to rename fields?

This solution allows us to make fields private which means that they're not accessible from outside of class code.